tox.ini

```diff
@@ -4,8 +4,9 @@ envlist = py27, py34
 [testenv]
 commands = {envpython} -m unittest discover
 deps =
```

Comment:
We should probably just link this with requirements.txt Originally, I was obsessed with having a minimal list of requirements for every environment, but now I think the technical debt of maintaining those isn't worth it.